src/content/PublicationsPageContent.ts

```diff
@@ -0,0 +1,34 @@
+import { stokes } from '../assets/assets.index';
+
+export type PubCardProps = {
+  title: string;
+  date: string;
+  text: string;
+  link: string;
+  imgSrc: string;
```

Comment:
Can you make this an actual react component/image instead of the path to the image? This way type checking will catch the lack of existence of a photo.

Reply:
I think leaving it as string works the same as react component/image too since in the end you'll need to input the path to the image in `src` attribute of `<img />`; therefore, it is still possible for there to be no photos.
for example:
```
const stokes = "";

<PubCard key={c.title} {...c} imgElement={<img src={stokes} />} />

<PubCard key={c.title} {...c} imgSrc={""} />
```

These two components will result the same. One way to fix this instead is to directly check the input:

```
const stokes = "";
const errorLink = "link to error image placeholder";

<PubCard key={c.title} {...c} imgElement={stoke ? <img src={stokes} /> : <img src={errorLink} />} />

<PubCard key={c.title} {...c} imgSrc={stokes ? stokes : errorLink} />
```

Reply:
There's also `onerror` attribute of `<img />` element that loads the image if the image in `src` is not loaded in properly:
- https://www.w3schools.com/jsref/event_onerror.asp
- https://medium.com/@webcore1/react-fallback-for-broken-images-strategy-a8dfa9c1be1e

Reply:
Super fair argument. Maybe in the future we incorporate an error image to help catch any issues. But for now it should be fine! Thanks!